pkgs/by-name/fl/flare-signal/package.nix

```diff
@@ -14,31 +16,56 @@
   gtksourceview5,
   rustPlatform,
   rustc,
-  appstream-glib,
+  appstream,
   blueprint-compiler,
   desktop-file-utils,
   wrapGAppsHook4,
 }:
 
+let
+  presage = fetchFromGitHub {
+    owner = "whisperfish";
+    repo = "presage";
+    # match with commit from Cargo.toml
+    rev = "123c1f926e359c21b34d099279ee8a92462ce96d";
```

Comment:
Use `jq` to compare this against the revision in e.g. `de.schmidhuberj.Flare.json` and exit if they don't match. See e.g. what deltachat-desktop does.